src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -188,13 +177,12 @@ object WorkflowDescriptor {
     }
   }
 
-  private def validateBackendOptions(id: WorkflowId, options: WorkflowOptions, backend: Backend): ErrorOr[WorkflowOptions] = {
+  private def validateBackendOptions(id: WorkflowId, options: WorkflowOptions): ErrorOr[WorkflowOptions] = {
     try {
-      backend.assertWorkflowOptions(options)
       options.successNel
     } catch {
       case e: Exception =>
-        s"Workflow $id has invalid options for backend ${backend.backendType}: ${e.getMessage}".failureNel
+        s"Workflow $id has invalid options: ${e.getMessage}".failureNel
```

Comment:
Also just a reminder that the backend-specifi runtime attribute checking needs to be put back in - both for workflow submission and workflow validation